### PR TITLE
fix(vtkOpenGLVolumeMapper): Use Actor key matrices for Volume positio…

### PR DESCRIPTION
**Pull request by llangholz89**
…ning

re #1061

### CODE REVIEW

**Comment by llangholz89** [comment]
fyi @jourdain and @martinken, this fixes the positioning issues I was seeing, passes all tests, and plays nice with existing examples.  However, I'm not sure that its 100% complete.  I'm wondering if I'm missing something with regards to the lighting/normals and would appreciate another set of eyes!

**Comment by jourdain** [comment]
@martinken should we merge?

**Comment by martinken** [comment]
+1

**Comment by llangholz89** [comment]
I was unsure whether any actor rotations would need to be incorporated into the `model.idxNormalMatrix` [here](https://github.com/Kitware/vtk-js/pull/1071/commits/9dcb0bda5be3cbb1c2cf96f6cb4c38def37f7062#diff-29461d7f642f91d985f615189f7f6500L586)

**Comment by martinken** [comment]
Yes merge please

**Comment by jourdain** [comment]
Will do once all the test come back green

**Comment by martinken** [comment]
Yes, you need something like
```
        const anorms = mat3.create();
        mat3.multiply(anorms, keyMats.normalMatrix, actMats.normalMatrix);
        program.setUniformMatrix3x3('normalMatrix', anorms);

```

**Comment by martinken** [comment]
But in this case just use it in the calc for idxNormalMatrix

**Comment by llangholz89** [comment]
I thought that may be the case... See the new commit

**Comment by martinken** [comment]
Once again I fail to submit my comments. Can you whip up a new topic to make the suggested change?  Thanks!

File: Sources/Rendering/OpenGL/VolumeMapper/index.js [hunk 1]
```diff
@@ -575,19 +579,17 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
...
 
     mat3.copy(model.idxNormalMatrix, model.currentInput.getDirection());
-    mat3.multiply(
-      model.idxNormalMatrix,
-      keyMats.normalMatrix,
-      model.idxNormalMatrix
-    );
+    const normMat = mat3.create();
+    mat3.multiply(normMat, keyMats.normalMatrix, actMats.normalMatrix);
+    mat3.multiply(model.idxNormalMatrix, normMat, model.idxNormalMatrix);
```
[the remark this answers is not in the document]
Comment (llangholz89) [reply Sources/Rendering/OpenGL/VolumeMapper/index.js]: That makes sense.  The `mcvc` mat4 created at the beginning could probably be stored on the model in the same way that `idxToView` and `idxNormalMatrix` are

**Comment by llangholz89** [comment]
Haha, sure thing.  I'll add a 'perf' tag to the new commit

**Pull request merged**